src/agenteval/score.py

```diff
@@ -162,25 +166,22 @@ def process_eval_logs(log_dir: str) -> tuple[list[TaskResult], bool]:
     results = []
     for task_name, log in logs.items():
         eval_spec = EvalSpec.from_eval_log(log)
-        try:
-            metrics = get_metrics(log)
-            if len(metrics) == 0:
-                raise ValueError(f"No metrics found for task {task_name}.")
-            model_usages = get_model_usages(log)
-            model_costs = [compute_model_cost(usages) for usages in model_usages]
-            has_model_usages = any(len(usages) > 0 for usages in model_usages)
-            results.append(
-                TaskResult(
-                    task_name=task_name,
-                    eval_spec=eval_spec,
-                    metrics=metrics,
-                    # Set to None to avoid incorrect pyarrow model usage type inference
-                    model_usages=model_usages if has_model_usages else None,
-                    model_costs=model_costs if has_model_usages else None,
-                )
+        metrics = get_metrics(log)
+        if len(metrics) == 0:
+            errors.append(f"No metrics found for task {task_name}.")
+            continue
+        model_usages = get_model_usages(log)
+        model_costs = [compute_model_cost(usages) for usages in model_usages]
+        has_model_usages = any(len(usages) > 0 for usages in model_usages)
+        results.append(
+            TaskResult(
+                task_name=task_name,
+                eval_spec=eval_spec,
+                metrics=metrics,
+                # Set to None to avoid incorrect pyarrow model usage type inference
+                model_usages=model_usages if has_model_usages else None,
+                model_costs=model_costs if has_model_usages else None,
             )
-        except ValueError as error:
-            had_errors = True
-            logger.exception(f"No metrics for {task_name}:")
+        )
 
-    return results, had_errors
+    return TaskResults(results=results, errors=errors)
```

Comment:
```suggestion
    return EvalLogProcessingResult(results=results, errors=errors)
```
(not sure exactly which it's supposed to be but TaskResults isn't imported in this file)
I daresay this highlights a weakness in the CI checks...